tests/CMakeLists.txt

```diff
@@ -536,7 +536,7 @@ if (UNIX)
           VERBATIM)
       add_custom_target("${kmod_name}"
           DEPENDS "${kmod_bin}")
-      add_dependencies(syscalls_unix "${kmod_name}")
+      #add_dependencies(syscalls_unix "${kmod_name}")
```

Comment:
Another mistakenly included change.  Is this that #2049?  Is it just missing `if (NOT X64)?`

Reply:
Done

Reply:
Yes it might, I had changed this locally to be able to build.